codalab/worker_manager/kubernetes_worker_manager.py

```diff
@@ -138,7 +138,7 @@ def start_worker_job(self) -> None:
         config: Dict[str, Any] = {
             'apiVersion': 'v1',
             'kind': 'Pod',
-            'metadata': {'name': worker_name},
+            'metadata': {'name': worker_name, 'labels': {'type': 'cl-worker'}},
```

Comment:
Nit: change 'type' to 'app'. I see that a lot more often (e.g. [here](https://kubernetes.io/docs/concepts/workloads/controllers/deployment/)).

Reply:
Done